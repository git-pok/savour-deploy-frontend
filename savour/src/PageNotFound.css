.PageNotFound {
    display: block;
    float: left;
    width: 100vw;
    height: 100vh;
    padding-top: 120px;
    background-color: rgb(37, 37, 37);
}

.PageNotFound-text {
    display: block;
    width: 460px;
    height: 180px;
    margin: 5px auto 5px auto;
    border-radius: 10px;
    border: 3px solid rgb(196, 126, 126);
    padding-top: 49px;
    background-color: white;
}

.PageNotFound-text p {
    display: inline-block;
    text-align: left;
    margin: 0px auto 20px auto;
}