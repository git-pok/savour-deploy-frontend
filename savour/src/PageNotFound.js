import CenterText from './CenterText.js';
import useToggleState from './hooks/useToggleState.js';
import './PageNotFound.css';

/**
 * PageNotFound
 * Props: none
 * Renders page not found.
*/
const PageNotFound = () => {
  return (
    <div className="PageNotFound">
      <div className="PageNotFound-text">
        <CenterText text={"Page Not Found"} color="black" fontSize={60} />
        <CenterText text={"Click back button or a menu tab!"} color="black" fontSize={23} />
      </div>
    </div>
  );
}

export default PageNotFound;