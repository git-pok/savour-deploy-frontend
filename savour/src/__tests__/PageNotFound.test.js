import PageNotFound from '../PageNotFound.js';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

test("renders PageNotFound without crashing", () => {
  render(
    <MemoryRouter>
      <PageNotFound />
    </MemoryRouter>
  );
});

test("renders snapshot without crashing", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <PageNotFound />
    </MemoryRouter>
  );

  expect(asFragment()).toMatchSnapshot();
});
