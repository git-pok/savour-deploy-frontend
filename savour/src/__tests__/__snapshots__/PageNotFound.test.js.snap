// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot without crashing 1`] = `
<DocumentFragment>
  <div
    class="PageNotFound"
  >
    <div
      class="PageNotFound-text"
    >
      <p
        style="color: black; font-size: 60px;"
      >
        Page Not Found
      </p>
      <p
        style="color: black; font-size: 23px;"
      >
        Click back button or a menu tab!
      </p>
    </div>
  </div>
</DocumentFragment>
`;
